chisel.yml

```diff
@@ -4,88 +4,139 @@ blake2:
     preset: "ewasm"
   trimexports:
     preset: "ewasm"
+  verifyimports:
+    preset: "ewasm"
+  verifyexports:
+    preset: "ewasm"
 
 bls12pairing:
   file: "target/wasm32-unknown-unknown/release/ewasm_precompile_bls12pairing.wasm"
   remapimports:
     preset: "ewasm"
   trimexports:
     preset: "ewasm"
+  verifyimports:
+    preset: "ewasm"
+  verifyexports:
+    preset: "ewasm"
```

Comment:
@jakelang this is already showing that we need a meta step which does all the translations and another one which does all the verification.

Reply:
not necessarily? I feel like this is simple enough. Yes it is a bit more verbose, but not exactly a hindrance.

Reply:
Imaging doing it for that many times I did...